### PR TITLE
Add relationships to the objects

### PR DESCRIPTION
**Pull request by DelhinRharl**
## In this pull request, 

- [ ] I added associations to the project's objects

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require './rental'
+class Book
+  attr_accessor :label, :author
+  attr_reader :rentals
```
Comment (Emmyn5600) [inline book.rb]: - [x] One of the instance variables is missing inside the setters and getters initialization. 

please add the `: title` to the attr_accessor and you can even add the `: rentals` to the attr_accessor to set has many rentals.

File: teacher.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 require './person'
+
 class Teacher < Person
   def initalize(age, specialization, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
```
Comment (Emmyn5600) [inline teacher.rb]: - [ ] When setting the classroom for a student it also adds it to the classrooms' students:

The class should be `class Student < Person` not `class Teacher < Person`, kindly implement that class.

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initalize(date)
+    @date = date
+    @person = person
+    person.rentals << self
+    @book = book
+    book.rental << self
```
Comment (Emmyn5600) [inline rental.rb]: - [ ] Here should be `book.rentals << self` not `book.rental`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @DelhinRharl 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:

- All linters are passing :heavy_check_mark:
- Good commit history :heavy_check_mark:
- Good PR description :heavy_check_mark:

## Required Changes :recycle:

- [ ] Check the comment under the review

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 2 above)
Comment (DelhinRharl) [reply teacher.rb]: kindly check the students.rb

File: rental.rb (hunk 3 above)
Comment (rloterh) [reply rental.rb]: This issue is well resolved! ✔️

File: teacher.rb (hunk 2 above)
Comment (rloterh) [reply teacher.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Affax, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**